src/db/types.rs

```diff
@@ -12,4 +12,16 @@ impl Feature {
     pub fn new(name: String, subfeatures: Vec<String>) -> Self {
         Feature { name, subfeatures }
     }
+
+    pub fn is_private(&self) -> bool {
+        self.name.starts_with('_')
+    }
+
+    pub fn is_default(&self) -> bool {
+        self.name == "default"
+    }
+
+    pub fn subfeature_len(&self) -> usize {
+        self.subfeatures.len()
+    }
```

Comment:
Is this function necessary? It seems like it would be more clear inlined.

Reply:
To be honest I didn't know that you can do something like that, thanks